host/xtest/xtest_10000.c

```diff
@@ -280,7 +280,7 @@ static const uint8_t pbkdf2_3_dkm[] = {
 #define pbkdf2_4_password pbkdf2_1_password
 #define pbkdf2_4_salt pbkdf2_1_salt
 #define pbkdf2_4_iteration_count 16777216
-static const uint8_t pbkdf2_4_dkm[] = {
+static const uint8_t pbkdf2_4_dkm[] __maybe_unused = {
```

Comment:
;)

Yep, basically the "host" (non-secure) part of `xtest` has full access to the TA API, which I don't think is a good thing (i.e., `#include <tee_internal_api.h>` makes no sense). Some trimming of what we're putting in `host_include` would be needed I suppose.